specification/cognitiveservices/data-plane/LUIS/Programmatic/v2.0/LUIS-Programmatic.json

```diff
@@ -4780,6 +5007,28 @@
           "type": "string"
         }
       },
+      "example": {
+        "name": "FlightClass",
+        "children": [ "Economy", "First" ]
+      }
+    },
+    "CompositeModelCreateObject": {
```

Comment:
`CompositeModelCreateObject` and `CompositeModelUpdateObject` are essentially the same; can we use one model for this (maybe `CompositeModel` or something?